### PR TITLE
Branch large start number

### LINKED ISSUE #119: [PE-D] Allowing extremely large number of questions for quiz
This is not so much a bug but perhaps a temporary design flaw since the total number of flashcards current available are around 150+ but number of questions allowed to be entered for quiz can be a billion i.e. 1000000000.

[WeebLingo 2021-04-03 14-39-37.mp4](https://raw.githubusercontent.com/jaredtengsw/ped/main/files/bb5ab1de-138e-42cf-9fda-2e670520a08a.mp4)

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: jaredtengsw/ped#2

### LINKED ISSUE #155: [PE-D] Incorrect error message for start
Number provided is positive but the error message says otherwise. This might be because very large numbers were not considered. Please take of this!

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: kumsssss/ped#12

### LINKED ISSUE #157: [PE-D] Creating a new error message for too large values of n for quiz questions
Could create a new exception when the numbers are too large as the current exception message states that the number of questions must be a positive integer when i entered a positive large value.

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: Jacob-109/ped#2

### LINKED ISSUE #161: [PE-D] Quiz questions limit not working as I thought it would
I tagged the first 4 flashcards with the tag insane and tried to start the quiz with n/5 which should result in a quiz with 5 questions consisting of cards with the tag insane. However, there was only 4 questions in the quiz hence i feel that either a exception should show stating that n is larger than the number of flashcards with the given tag or that the quiz should have 5 questions( e.g. after going through the first 4 then it will give a random one).

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: Jacob-109/ped#6

### PR DESCRIPTION
**Pull request by xinweit**
Add exception for number to be < 2^32 and number to be <= number of flashcards in database

### CODE REVIEW

File: src/main/java/seedu/weeblingo/logic/commands/StartCommand.java [hunk 1]
```diff
@@ -33,6 +33,14 @@
     public static final String MESSAGE_INVALID_NUMBER_OF_QUESTIONS = "Oops! Number of questions must "
             + "be a positive integer!";
 
+    public static final String MESSAGE_NUMBER_TOO_LARGE = "The number you specified is too large! "
+            + "Please enter a number that is smaller than 2^32.";
```
Comment (Yanneko) [inline src/main/java/seedu/weeblingo/logic/commands/StartCommand.java]: Normal users probably will have problems with what 2^32 is.

File: src/main/java/seedu/weeblingo/logic/parser/StartCommandParser.java [hunk 2]
```diff
@@ -29,8 +29,9 @@ public StartCommand parse(String args) throws ParseException {
         } else {
             try {
                 numberOfQuestions = Integer.parseInt(stringNumberOfQuestions.get());
+                System.out.println(numberOfQuestions);
```
Comment (Yanneko) [inline src/main/java/seedu/weeblingo/logic/parser/StartCommandParser.java]: Probably shouldn't be leaving debugging lines in our actual code.

Comment (xinweit) [reply src/main/java/seedu/weeblingo/logic/parser/StartCommandParser.java]: thanks for spotting. removed

File: src/main/java/seedu/weeblingo/logic/commands/StartCommand.java (hunk 1 above)
Comment (xinweit) [reply src/main/java/seedu/weeblingo/logic/commands/StartCommand.java]: changed to 1 million maximum

**Review by Jellybeano — APPROVED** [review]
lgtm~

**Pull request merged**